### PR TITLE
Update for 2020 applications

### PR DESCRIPTION
**Pull request by pblouw**
This PR updates the summer school page for next year, and links to a new application form. I've tested a build locally, and it looks fine.

### CODE REVIEW

**Comment by pblouw** [comment]
Just adding a quick ping to move this up the queue - it would be great to get it merged in so we can open up for applications to the summer school before the holiday break.

File: summer-school.md [hunk 1]
```diff
@@ -15,18 +15,18 @@ style="width: 100%;"/>]({{ site.baseurl }}/img/summerschool-campus.png)
 **Nengo with Loihi:**
```
Comment (drasmuss) [inline summer-school.md]: Do we still want "Nengo with Loihi" to be the main heading here? Or just something more general about Nengo/neuromorphics?

File: summer-school.md [hunk 2]
```diff
@@ -15,18 +15,18 @@ style="width: 100%;"/>]({{ site.baseurl }}/img/summerschool-campus.png)
...
+Waterloo is excited to announce our 7th annual Nengo summer school
 on large-scale brain modelling and neuromorphic computing. As a
-special feature of this year’s school, participants will be given
+recurring special feature of the school, participants will be given
 on-site access to [Loihi](https://ieeexplore.ieee.org/document/8259423),
 a neuromorphic research chip developed by Intel Labs. Other
 neuromorphic devices likely to be available at the summer school
 include Stanford’s mixed digital/analog chip
 [Braindrop](https://web.stanford.edu/group/brainsinsilicon/),
 and [SpiNNaker](http://apt.cs.manchester.ac.uk/projects/SpiNNaker/).
```
Comment (drasmuss) [inline summer-school.md]: Could mention the FPGA here as well?

**Comment by pblouw** [comment]
Good suggestions! I pushed a fixup that addresses both - feel free to let me know if there's anything else that could use a tweak.

**Comment by drasmuss** [comment]
Looks good, will merge now!

**Pull request closed without merging**